R/init.R

```diff
@@ -72,10 +72,15 @@ copy_assets <- function(pkg = ".") {
 
   # Copy assets from package
   if (!is.null(template$package)) {
-    copy_asset_dir(
-      pkg,
-      path_package_pkgdown(template$package, bs_version = NULL, "assets")
-      )
+    assets <- path_package_pkgdown(
+      template$package,
+      bs_version = get_bs_version(pkg),
+      "assets"
+    )
+
+    if (length(assets) > 0) {
```

Comment:
The current code was assuming a template package had to provide assets, which this PR fixes.

Reply:
I think this check should go in `copy_asset_dir()`